Dockerfiles/WebApp

```diff
@@ -10,28 +10,36 @@ ARG APP=target/webapp
 ENV JAVA_HOME /opt/java/openjdk
 COPY --from=jdk $JAVA_HOME $JAVA_HOME
 
-# Will be replaced on Jetty in 5.26
-# Copy Tomcat
-ENV CATALINA_HOME /usr/local/tomcat
-COPY --from=tomcat:9-jre11-temurin-focal $CATALINA_HOME $CATALINA_HOME
+ENV OPENL_APP /openl-app
+ENV JETTY_HOME $OPENL_APP
+ENV JETTY_BASE $OPENL_APP
+ENV OPENL_LIB /openl-lib
 
-# Define executables
-ENV PATH $CATALINA_HOME/bin:$JAVA_HOME/bin:$PATH
-
-WORKDIR $CATALINA_HOME
-
-# Fix SEVERE error while running image via non-root user
-RUN mkdir -p conf/Catalina/localhost
-
-# Install fonts required for Apache POI (export into Excel with autowidth of columns)
+COPY --from=jetty:9-jre11-slim /usr/local/jetty $JETTY_HOME
+COPY --from=jetty:9-jre11-slim /var/lib/jetty $JETTY_BASE
 RUN set -eux ; \
+\
+# Create start file for Jetty with configuration options
+    mkdir $OPENL_LIB; \
+    ln -s $OPENL_LIB $JETTY_HOME/lib/ext; \
+    echo '#!/usr/bin/env bash\n\n\
+export JAVA_OPTIONS=-Xms32m\ -XX:MaxRAMPercentage=90.0\ $JAVA_OPTS\ $JAVA_OPTIONS\n\
+source $JETTY_HOME/bin/jetty1.sh run\n' >> $JETTY_HOME/bin/start.sh; \
+    chmod +x $JETTY_HOME/bin/start.sh; \
+\
+# Install fonts required for Apache POI (export into Excel with autowidth of columns)
+    apt-get update; \
+    chmod +x ${JETTY_HOME}/bin/jetty.sh; \
     \
-    apt-get update ; \
     apt-get install -y --no-install-recommends \
     fontconfig ; \
     rm -rf /var/lib/apt/lists/*
 
+# Define executables
+ENV PATH $JETTY_HOME/bin:$JAVA_HOME/bin:$PATH
+
 ENV OPENL_HOME /openl
+ENV OPENL_HOME_SHARED /openl
```

Comment:
What is the difference between OPENL_HOME and OPENL_HOME_SHARED? Why do you need both of them?

Reply:
According to @yurkom it should have both: one for local, one for shared with other nodes.